[2024-06-29T00:48:37.049+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-29T00:48:37.060+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: create_dag_with_python_operator_v06.first_task scheduled__2024-06-28T00:00:00+00:00 [queued]>
[2024-06-29T00:48:37.063+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: create_dag_with_python_operator_v06.first_task scheduled__2024-06-28T00:00:00+00:00 [queued]>
[2024-06-29T00:48:37.063+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-29T00:48:37.068+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): first_task> on 2024-06-28 00:00:00+00:00
[2024-06-29T00:48:37.070+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12110) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-29T00:48:37.070+0000] {standard_task_runner.py:63} INFO - Started process 12111 to run task
[2024-06-29T00:48:37.070+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'create_dag_with_python_operator_v06', 'first_task', 'scheduled__2024-06-28T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/creat_dag_with_python_operator.py', '--cfg-path', '/tmp/tmppk0n5ukl']
[2024-06-29T00:48:37.071+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask first_task
[2024-06-29T00:48:37.085+0000] {task_command.py:426} INFO - Running <TaskInstance: create_dag_with_python_operator_v06.first_task scheduled__2024-06-28T00:00:00+00:00 [running]> on host 6f85ce718b3c
[2024-06-29T00:48:37.110+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nash5657' AIRFLOW_CTX_DAG_ID='create_dag_with_python_operator_v06' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-28T00:00:00+00:00'
[2024-06-29T00:48:37.110+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-29T00:48:37.122+0000] {logging_mixin.py:188} INFO - Hello World from Nash5657 and I am 19 years old
[2024-06-29T00:48:37.123+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-29T00:48:37.123+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-29T00:48:37.125+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=create_dag_with_python_operator_v06, task_id=first_task, run_id=scheduled__2024-06-28T00:00:00+00:00, execution_date=20240628T000000, start_date=20240629T004837, end_date=20240629T004837
[2024-06-29T00:48:37.136+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-29T00:48:37.141+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-29T00:48:37.142+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
